parsers/ajenti.py

```diff
@@ -18,29 +18,33 @@
 from signalr import Connection
 
 ZONE_PARAMS = {
+    # King's Island
```

Comment:
* King Island, not King's Island

Reply:
Thanks, good catch!

Fixes typo
```suggestion
    # King Island
```